src/pages/on-chain-governance/_components/ProposalCard.tsx

```diff
@@ -70,216 +69,178 @@ export enum ProposalDisplayName {
+                    isOpenProposalsClicked
+                      ? "/blocks"
+                      : `/blocks/${proposal.cycleEndHeight}`
+                  }
+                >
+                  {`Block ${proposal.cycleEndHeight}`}
+                </a>
+              </Link>
+              <div>
+                <div className="text-dark-gray-50 font-semibold text-sm text-right dark:text-dark-gray-900">
+                  {`~ ${cycleEndTime}`}
+                </div>
               </div>
-            </>
-          )}
-        </div>
-        <div />
-      </div>
-      <div className="group md:hidden block hover:bg-primary-50 dark:hover:bg-gray-600">
-        <div
-          className={classNames(
-            "grid pt-5 pb-6 px-6 gap-y-6 border-b",
-            currentStage === votingStages.vote
-              ? "grid-cols-2 grid-rows-[44px_minmax(0px,_1fr)_44px]"
-              : "grid-cols-2 grid-rows-[44px_minmax(0px,_1fr)_24px]"
-          )}
-        >
-          <div className="flex flex-col gap-y-1">
-            <div className=" text-xs text-gray-500">
-              {OnChainGovernanceTitles.nameOfProposalTitle}
-            </div>
-            <div className="text-gray-900 group-hover:text-primary-500">
-              {proposal.title}
-            </div>
-          </div>
-          <div className="flex flex-col gap-y-1 text-right">
-            <div className="text-xs text-gray-500">
-              {OnChainGovernanceTitles.typeTitle}
-            </div>
-            <div className="text-gray-900 group-hover:text-primary-500">
-              {ProposalDisplayName[proposal.type]}
-            </div>
-          </div>
-          <div className="row-start-2 flex flex-col gap-y-1">
-            <div className=" text-xs text-gray-500">
-              {OnChainGovernanceTitles.proposerId}
-            </div>
-            <div className="text-gray-900 group-hover:text-primary-500">
-              <TextTruncate text={proposal.proposalId} width="w-full" />
             </div>
           </div>
 
-          <div
-            className={classNames(
-              "flex flex-row gap-x-[27px]",
-              currentStage === votingStages.vote ? "row-start-4" : "row-start-3"
-            )}
-          >
+          <div className="flex flex-row align-middle">
+            <div className="text-sm text-gray-500 grow dark:text-dark-gray-500">
+              Discussion
+            </div>
             <a
+              href={proposal.context}
               onClick={(e) => {
                 e.stopPropagation();
               }}
-              className="flex flex-row items-center gap-x-2 text-[#4A72DA] hover:underline cursor-pointer"
-              href={proposal.context}
             >
-              {OnChainGovernanceTitles.github}
-              <IoMdOpen size={24} />
+              <div className="flex flex-row font-semibold items-center gap-x-2 text-sm text-dark-gray-50 dark:text-dark-gray-900">
+                <AiFillGithub size={20} />
```

Comment:
can we change it to slightly bigger to compensate the empty spaces of the icon, then need to adjust the padding accordingly

Reply:
Icon size was adjusted initial icon size is ~16px updated to 20

fixed padding between `Github` and Icon